docs/reference/run.md

```diff
@@ -1605,6 +1605,7 @@ the `USER` instruction by passing the `-u` option.
     --user=[ user | user:group | uid | uid:gid | user:gid | uid:group ]
 
 > **Note:** if you pass a numeric uid, it must be in the range of 0-2147483647.
+> If you pass a user name, the user must be already exist in the container.
```

Comment:
Sorry for the long delay; think this PR dropped of my radar; I see a small grammatical error; `be already exist` should probably be `must already exist`, but perhaps we can simplify a bit;

```suggestion
> If you pass a username, the user must exist in the container.
```

Reply:
Thanks, it has been edited as suggested, please review again.